src/components/Search/SearchInput.tsx

```diff
@@ -82,6 +82,22 @@ type SearchInputListener = {
   onStateChange: (isOpen: boolean) => void;
 };
 
+function parseSuggestions(suggestions: SearchAutoSuggestionResultItem[] = []) {
+  return suggestions.map((suggestion) => {
+    const [allTitle, bundleTitle, abstract] = suggestion.term.split(AUTOSUGGEST_TERM_DELIMITER);
+    const url = new URL(suggestion.payload);
```

Comment:
Wouldn't it be better to directly destruct the `pathname` here? `const { pathname} = ...`

Reply:
It's a class, not an object. Destructing classes might be dangerous.